resources/config.yml

```diff
@@ -1,7 +1,12 @@
 version: 2
```

Comment:
Changes to config require a bump, and the validation updated and a migration function added to https://github.com/DiscordBot-PMMP/DiscordAccount/blob/3e0a3c335fabffb2a9c90a7adf1ded59c50b0aa4/src/ConfigUtils.php#L26